cli/configuration.schema.js

```diff
@@ -0,0 +1,59 @@
+export default {
+  $schema: 'http://json-schema.org/schema#',
+  title: 'Configuration',
+  type: 'object',
+  required: ['baseUrl', 'jwt', 'mongo', 'storage'],
+  properties: {
+    baseUrl: {
+      type: 'string',
+      description: 'Server base url',
+    },
+    jwt: {
+      type: 'object',
+      required: ['type', 'secret'],
+      properties: {
+        type: {
+          type: 'string',
+          enum: ['hmac'],
+        },
+        secret: {
+          type: 'string',
```

Comment:
Let's add minLength (8 chars should be the minimum allowed length) and maxLength here:

```
secret: {
  type: 'string',
  minLength: 8,
  maxLength: 255
}
```

Reply:
Added minLength